server/odc-service/src/main/java/com/oceanbase/odc/service/session/ConnectConsoleService.java

```diff
@@ -303,9 +312,14 @@ public SqlAsyncExecuteResp streamExecute(@NotNull String sessionId,
                 Objects.nonNull(request.getContinueExecutionOnError()) ? request.getContinueExecutionOnError()
                         : userConfigFacade.isContinueExecutionOnError();
         boolean stopOnError = !continueExecutionOnError;
-        OdcStatementCallBack statementCallBack = new OdcStatementCallBack(sqlTuples, connectionSession,
-                request.getAutoCommit(), queryLimit, stopOnError, executeContext);
-
+        OdcStatementCallBack statementCallBack = null;
```

Comment:
this method is designed for sql execution, don't add any non-related logic here, please

Reply:
ob oracle implement pl editing functionality in that.